pallets/parachain-staking/src/lib.rs

```diff
@@ -2116,40 +2238,65 @@ pub mod pallet {
 	{
 		/// Compute coinbase rewards for block production and distribute it to
 		/// collator's (block producer) and its delegators according to their
-		/// stake.
+		/// stake and the current InflationInfo.
+		///
+		/// The rewards are split between collators and delegators with
+		/// different reward rates and maximum staking rates. The latter is
+		/// required to have at most our targeted inflation because rewards are
+		/// minted. Rewards are immediately available without any restrictions
+		/// after minting.
+		///
+		/// If the current staking rate is below the maximum, each collator and
+		/// delegator receives the corresponding `reward_rate * stake /
+		/// blocks_per_year`. Since a collator can only author blocks every
+		/// `MaxSelectedCandidates` many rounds, we multiply the reward with
+		/// this number. As a result, a collator who has been in the set of
+		/// selected candidates, eventually receives `reward_rate * stake` after
+		/// one year.
+		///
+		/// However, if the current staking rate exceeds the max staking rate,
+		/// the reward will be reduced by `max_rate / current_rate`. E.g., if
+		/// the current rate is at 50% and the max rate at 40%, the reward is
+		/// reduced by 20% by multiplying the reward with 80%.
```

Comment:
I would just say that the reward is reduced by 20%.

```suggestion
		/// reduced by 20%.
```